funsor/jax/ops.py

```diff
@@ -147,6 +147,11 @@ def _log(x):
     return np.log(x)
 
 
+@ops.mean.register(array, (int, type(None)))
+def _mean(x, dim):
```

Comment:
Note there's no way to specify `dim` or `ddof` if you use the simple op factory `mean = make_op(np.mean)`. If you want fancier behavior you'd need to define a
```py
class MeanOpMeta(CachedOpMeta):
    ...
class MeanOp(Op, metaclass=MeanOpMeta):
    def __init__(self, axis=None, keepdims=False):
        ...
    ...
mean = MeanOp()  # default behavior
```